src/wrappers/themis/ruby/lib/rubythemis.rb

```diff
@@ -360,24 +410,31 @@ def Ssign(*args)
   deprecate :Ssign, :s_sign, 2018, 6
 
   def s_sign(private_key, message)
+    if not valid_key(private_key)
```

Comment:
technically speaking for user both these checks are the same -- wrong / empty / bad key.

Reply:
I would rather prefer to merge these checks in one, or use "Secure Message: private key used instead of public" as second error message

Reply:
There's a difference between "using an incorrect key" and "using a key incorrectly". If we get to the second check we are sure that the key is at least valid so we should not tell the user that their key is garbled or something.

I guess I'll reuse the same messages as for the encrypt/decrypt mode.